client/client-multi/client-c/build.sh

```diff
@@ -168,7 +108,10 @@ case "$cmd" in
         UNITTESTS_COMPILE=1
```

Comment:
Why don't use them? It's pretty useful not to compile unit tests when there is no need.